### ISSUE TITLE
`incorrect-package.xsl` fails on one-letter names

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#226]
This is my code:

```
+package a.b.c

[] > app
  42 > @
```

`incorrect-package.xsl` says this:

```
The format of the +package meta is wrong: "a.b.c"
```

This definitely is a false positive.
